webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala

```diff
@@ -644,11 +646,16 @@ object RudderParsedProperties {
   val RUDDER_BATCH_REPORTSCLEANER_COMPLIANCE_DELETE_TTL =
     config.getInt("rudder.batch.reportscleaner.compliancelevels.delete.TTL") // AutomaticReportsCleaning.defaultDeleteTTL
   val RUDDER_BATCH_REPORTSCLEANER_LOG_DELETE_TTL = {
-    try {
+    (Try {
+      config.getString("rudder.batch.reportscleaner.deleteLogReport.TTL")
```

Comment:
it should be `rudder.batch.reportsCleaner.deleteLogReport.TTL` (with 'C' for cleaner)